File: api-code-examples/nodejs/authors-create.js
import { Iroh } from '@number0/iroh'

const node = await Iroh.memory()
const nodeId = await node.node.nodeId()
console.log(`Started iroh node: ${nodeId}`)

const author = await node.authors.create()
console.log(`Created author: ${author.toString()}`)

File: api-code-examples/rust/authors-create.rs
#[tokio::main]
async fn main() -> anyhow::Result<()> {
    let node = iroh::node::Node::memory().spawn().await?;
    let node_id = node.net().node_id().await?;
    println!("Started Iroh node: {node_id}");

    let author = node.authors().create().await?;
    println!("Created author: {author}");

    Ok(())
}
